core/arch/aarch64/instr_create.h

```diff
@@ -47,18 +47,33 @@
  */
 #define FSZ_HALF 1
 
+/**
+ * Operand indication half-precision floating point vector elements.
```

Comment:
s/indication/indicating/

Maybe also:
s/elements./elements for the other operands of the containing instruction./